docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -232,4 +339,19 @@ GET books/_search
   }
 }
 ----
-// TEST[continued]
\ No newline at end of file
+// TEST[continued]
+
+[discrete]
+[[getting-started-delete-indices]]
+=== Delete index
+
+Delete indices using the <<indices-delete-index,`DELETE` index API>>.
```

Comment:
why would you want to delete an index? might want a use case here

Reply:
not critical but feels helpful and like a natural bookend in the spirit of CRUD

The main use case for the tutorial reader is to start from scratch if an index ever gets funky

I'll add an _optional_ tag.

Happy to delete this if it's overkill tho.